engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpression.scala

```diff
@@ -206,7 +206,9 @@ object SpelExpressionParser extends LazyLogging {
               strictTypeChecking: Boolean,
               imports: List[String],
               flavour: Flavour,
-              strictMethodsChecking: Boolean)
+              strictMethodsChecking: Boolean,
+              staticMethodInvocationsChecking: Boolean,
+              typeDefinitionSet: TypeDefinitionSet = TypeDefinitionSet(Set()))
```

Comment:
I don't think this default helps - in all reasonable cases we should pass explicit list

Reply:
Changed it, thanks